packages/devextreme/testing/tests/DevExpress.ui.widgets/scrollableParts/scrollable.api.tests.js

```diff
@@ -788,6 +788,27 @@ QUnit.test('scrollTo should not reset unused position', function(assert) {
     assert.equal(scrollable.scrollTop(), 40, 'top position set');
 });
 
+QUnit.test('_onVisibilityChanged option should allow to set custom position', function(assert) {
```

Comment:
[actually on this level of testing it's enough to check that passed function is called, no need to make scenario more specific, but ok if u like it]